src/Controller/EnMarche/MembershipController.php

```diff
@@ -28,19 +29,28 @@ class MembershipController extends Controller
     /**
      * This action enables a guest user to adhere to the community.
      *
-     * @Route("/adhesion/{uuid}", name="app_register_fully")
+     * @Route("/adhesion", name="app_register_fully")
      * @Method("GET|POST")
+     *
+     * @Security("is_granted('ROLE_USER')")
      */
-    public function finaliseRegistrationAction(Request $request, Adherent $adherent): Response
+    public function finaliseRegistrationAction(Request $request): Response
```

Comment:
not related to your PR, but typo: finalize (we use American English).

Reply:
I'll rename it.